code/controllers/subsystem/logging.dm

```diff
@@ -0,0 +1,57 @@
+SUBSYSTEM_DEF(logging)
+	name = "Logging"
+	priority = INIT_ORDER_LOGGING
+	flags = SS_NO_FIRE
+	var/list/datum/log_record/logs	// Assoc list of assoc lists (ckey, (log_type, list/logs))
+
+/datum/controller/subsystem/logging/Initialize()
+	LAZYINITLIST(logs)
```

Comment:
If LAZYINITLIST is just `if (!L) L = list()`, why not just initialise the list on line 5, then you don’t even need the SS initialiser.

Reply:
Made this incase a log is made before the system is initliased. I'll change this though since it now inits as one of the earliest systems.